src/app/(proper_react)/layout.tsx

```diff
@@ -19,18 +22,74 @@ export default async function Layout({ children }: { children: ReactNode }) {
 
   return (
     <L10nProvider bundleSources={l10nBundles}>
-      <ControlsWrapper session={session}>
-        <div className={styles.page}>{children}</div>
-        <footer className={styles.footer}>
-          <a href="https://www.mozilla.org" target="_blank">
-            <Image
-              src={mozillaLogo}
-              width={100}
-              alt={l10n.getString("mozilla")}
-            />
-          </a>
-        </footer>
-      </ControlsWrapper>
+      <MobileShell session={session}>
+        <div className={styles.wrapper}>
+          <nav className={styles.mainMenu}>
+            <Link href="/" className={styles.homeLink}>
+              <Image
+                src={monitorLogo}
+                alt={l10n.getString("main-nav-link-home-label")}
+                width={170}
+              />
+            </Link>
+            <ul>
+              {/* Note: If you add elements here, also add them to <MobileShell>'s navigation */}
+              <li>
+                <PageLink
+                  href="/redesign/user/dashboard"
+                  activeClassName={styles.isActive}
+                >
+                  {l10n.getString("main-nav-link-dashboard-label")}
+                </PageLink>
+              </li>
+              <li>
+                <a
+                  href="https://support.mozilla.org/kb/firefox-monitor-faq"
+                  title={l10n.getString("main-nav-link-faq-tooltip")}
+                >
+                  {l10n.getString("main-nav-link-faq-label")}
+                </a>
+              </li>
+            </ul>
+          </nav>
+          <div className={styles.content}>
+            <div className={styles.page}>{children}</div>
+            <footer className={styles.footer}>
+              <a
+                href="https://www.mozilla.org"
+                className={styles.mozillaLink}
+                target="_blank"
+              >
+                <Image
+                  src={mozillaLogo}
+                  width={100}
+                  alt={l10n.getString("mozilla")}
+                />
+              </a>
+              <ul className={styles.externalLinks}>
```

Comment:
Q: Should these be in a `<nav>` wrapper? Or is it not valid to have multiple sets `<nav>` tags in headers and footers?

Reply:
Multple `<nav>` tags [are valid](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nav#usage_notes), as I understand it:

> A document may have several [<nav>](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nav) elements, for example, one for site navigation and one for intra-page navigation. [aria-labelledby](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-labelledby) can be used in such case to promote accessibility, see [example](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements#labeling_section_content).

But [supposedly](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nav#usage_notes) not in footers:

> It's not necessary for all links to be contained in a <nav> element. <nav> is intended only for a major block of navigation links; typically the [<footer>](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/footer) element often has a list of links that don't need to be in a [<nav>](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nav) element.